src/model/article-history.v1.yaml

```diff
@@ -1,25 +1,41 @@
 $schema: http://json-schema.org/draft-04/schema#
 title: Article history
-type: object
-properties:
-    received:
-        type: string
-        format: date
-    accepted:
-        type: string
-        format: date
-    versions:
-        description: List of article versions, sorted by version in descending order
-        type: array
-        items:
-          - oneOf:
-              - $ref: ../snippets/article-poa.v1.yaml
-              - $ref: ../snippets/article-vor.v1.yaml
-        minItems: 1
-dependencies:
-    received:
-      - accepted
-    accepted:
-      - received
-required:
-    - versions
+anyOf:
+-
+    type: object
+    properties:
+        received:
+            type: string
+            format: date
+        accepted:
+            type: string
+            format: date
+        versions:
+            description: List of article versions, sorted by version in descending order
+            type: array
+            items:
+              - oneOf:
+                  - $ref: ../snippets/article-poa.v1.yaml
+                  - $ref: ../snippets/article-vor.v1.yaml
+            minItems: 1
+    dependencies:
+        received:
+          - accepted
+        accepted:
+          - received
+    required:
+        - versions
+-
+    type: object
+    properties:
+        type:
+            type: string
+            enum:
+                - preprint
```

Comment:
> The content-type/date-type values we are going to use are:
preprint
accepted-manuscript
version-of-record
corrected-version-of-record

but not in this round of changes